### PR TITLE
Changes production time zone

### PR DESCRIPTION
**Pull request by j311y**
Small change to env/production to get Travis CI passing

### CODE REVIEW

**Review by chrisjmit — COMMENTED** [review]
:shipit:

File: db/schema.rb [hunk 1]
```diff
@@ -12,6 +12,18 @@
 
 ActiveRecord::Schema.define(version: 20170530143955) do
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "comments", force: :cascade do |t|
+    t.string "commenter"
```
Comment (dbugsy) [inline db/schema.rb]: This should probably be a foreign key of a user

**Review by dbugsy — CHANGES_REQUESTED** [review]

**Comment by JessicaBarclay** [comment]
:shipit:

File: db/schema.rb (hunk 1 above)
Comment (j311y) [reply db/schema.rb]: I don't know why this was included. Must of been changed between me pulling and pushing?

**Comment by Robert-G-J** [comment]
Hey @jonathanelliot, I pulled the branch and am checking it over. How can I check that Travis is passing? :)

File: db/schema.rb (hunk 1 above)
Comment (j311y) [reply db/schema.rb]: I'm not really sure how to resolve this. I didn't make these changes.

Comment (Robert-G-J) [reply db/schema.rb]: The comments branch was merged this morning- most definitely that I think

**Comment by j311y** [comment]
Hey @Robert-G-J. Go here: https://travis-ci.org/makersacademy/acebook-remote-april-2017

**Review by milesillsley — APPROVED** [review]
looks good to me :shipit:

**Comment by dbugsy** [comment]
:shipit: :shipit:

**Review by dbugsy — APPROVED** [review]

**Pull request merged**